app/package.json

```diff
@@ -42,7 +41,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-i18next": "11.7.0",
-    "react-router": "5.2.0",
```

Comment:
Bumped `react-router` library version and added the `*-dom` library as dependency.